app/components/filter-checkbox.js

```diff
@@ -4,4 +4,5 @@ import { argument } from '@ember-decorators/argument';
 export default class FilterCheckboxComponent extends Component {
   @argument value = '';
   @argument currentValues = [];
+  @argument classPrefix = null;
```

Comment:
Looks good - maybe this default should be a blank string since that’s what’s expected from it?